example.py

```diff
@@ -1,6 +1,6 @@
 import bdfparser as bp
 
-bpo = bp.BdfParser('example_fonts/bdf/SimSun-14.bdf')
+bpo = bp.BdfParser('example_fonts/bdf/unifont-9.0.06.bdf')
```

Comment:
I changed this to a font in the example_fonts directory.  The "SimSun-14.bdf" font file isn't included in the repo.

Reply:
Yea, I think initially I put SimSun and other non free font here. But then I decided to move them to [another repo](https://github.com/might-and-magic/fnt-generator) of mine, only hosting FOSS fonts in this repo. But I forgot to change "SimSun-14.bdf" in example.py and README.md. Thx. I'll merge your PR and change README.md later.